roosevelt.js

```diff
@@ -62,7 +63,8 @@ module.exports = function (params) {
 
   logger.log('💭', `Starting ${appName} in ${appEnv} mode...`.bold)
 
-  // let's try setting up the servers with user-supplied params
+  buildScanner(app)
```

Comment:
i think we should move this later in the execution somewhere after compileJs so that old files have a chance to be overwritten (and thus their dates updated) first